specification/repository.md

```diff
@@ -81,9 +81,12 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
     changes that have not yet been released.
   - The file MUST be in reverse chronological order, with the most recent
     releases at the top of the file, after the `Unreleased` section.
+  - Each release MUST contain a list of relevant changes, including those from upstream.
   - Each release SHOULD be separated by a line separator (`---`) from the other relases.
   - Each release SHOULD contain separate sections for each major functionality
     area (if applicable).
+  - Each change MUST bear a label that indicates where the change is coming from:
+    `(Contrib)`, `(Core)`, or `(Splunk)`.
```

Comment:
The labels don't really match the instrumentation repos: should it be (SDK), (Instrumentation), (Splunk)? Or perhaps we should allow any names, as long as they make sense within a particular project?

Reply:
> not sure about APM distros.

It depends. Java has SDK, Instrumentation and Contrib (yes, there's 3 of them); .NET has just the SDK + Instrumentation (I think, correct me if I'm wrong); Python/Node have Core/SDK & Contrib.

Reply:
That's OK, the labels are just examples.